applications/GeoMechanicsApplication/tests/cpp_tests/custom_processes/test_apply_c_phi_reduction_process.cpp

```diff
@@ -22,32 +21,27 @@ using namespace Kratos;
 
 namespace
 {
-ModelPart& SetGeometryAndMesh(Model& rModel)
+ModelPart& CreateModelPartWithElements(Model& rModel)
 {
     auto& result = rModel.CreateModelPart("dummy");
+    result.CreateNewProperties(0);
```

Comment:
Wouldn't it be clearer to store the return in a local variable and then pass this variable to the element constructors?
```suggestion
    auto p_properties = result.CreateNewProperties(0);
```

Reply:
Agreed, done